### PR TITLE
added section for command line utilities needed

### PR DESCRIPTION
**Pull request by davidyuanyue**
added section for command line utilities needed and where to download them

### CODE REVIEW

**Comment by sahirmemon** [comment]
These are really great. They're also part of the Backend Onboarding Confluence doc (see section regarding dotfiles) here https://pagerinc.atlassian.net/wiki/spaces/EN/pages/73596957/Backend+Onboarding.

File: backend/README.md [hunk 1]
```diff
@@ -75,6 +75,39 @@ Use your Pager credentials to login.
...
+These are the command line utilities you will likely to need to do your job
+
+**brew:** <https://brew.sh/>\
+If you use Mac, you are likely to need brew to install other ulitities
+
+**node:** <https://nodejs.org/en/download/>\
+Node js is the main platform we are using now
+
+**git:**
+Don't need to download anything, simply run xcode-select --install , git will be installed
```
Comment (cilindrox) [inline backend/README.md:90]: This installs an old version of git - for security and latest features, IMO we should stick to homebrew/git

Comment (DiegoRBaquero) [reply backend/README.md:90]: I think you need both? But I agree, brew one is more updated

Comment (davidyuanyue) [reply backend/README.md]: good catch, modified accordingly

Comment (cilindrox) [reply backend/README.md]: ditto - you need xcode utils to get homebrew, but after that it's advisable to replace the binary in your PATH with the one from the homebrew formula

**Pull request merged**